src/js/components/Layer/LayerContainer.js

```diff
@@ -11,6 +12,7 @@ import { defaultProps } from '../../default-props';
 import { FocusedContainer } from '../FocusedContainer';
 import { Keyboard } from '../Keyboard';
 import { backgroundIsDark, findVisibleParent } from '../../utils';
+import { PortalContext } from './PortalContext';
```

Comment:
Would it make sense to create a PortalContext util that will be shared between Layer and DropContainer?